data/json/items/magazine/shot.json

```
@@ -32,6 +32,39 @@
     "reload_time": 130,
     "flags": [ "MAG_BULKY" ]
   },
+  {
+    "id": "USAS10mag",
+    "type": "MAGAZINE",
+    "name": "USAS-12 box magazine",
+    "description": "A removable plastic magazine for the USAS-12 shotgun.  Holds 10 rounds.",
+    "weight": "750 g",
+    "volume": "360 ml",
+    "price": 4500,
+    "material": "plastic",
+    "symbol": "#",
+    "color": "dark_gray",
+    "ammo_type": "shot",
+    "capacity": 10,
+    "reliability": 9,
+    "flags": [ "MAG_COMPACT" ]
+  },
+  {
+    "id": "USAS20mag",
+    "type": "MAGAZINE",
+    "name": "USAS-12 drum magazine",
+    "description": "A removable plastic magazine for the USAS-12 shotgun.  Holds 20 rounds.",
+    "weight": "1 g",
```

```suggestion
    "weight": "1 kg",
```
Probably a typo? Can't have 1 gram magazine.

---

Ah thanks. it was a placeholder. I'll get that fixed.